app/build.gradle

```diff
@@ -268,9 +281,9 @@ dependencies {
     implementation "androidx.lifecycle:lifecycle-viewmodel-compose:$lifecycleVersion"
     implementation "androidx.lifecycle:lifecycle-extensions:2.2.0"
     implementation "androidx.paging:paging-runtime-ktx:$pagingVersion"
-    implementation 'androidx.activity:activity-ktx:1.5.1'
-    implementation 'androidx.activity:activity-compose:1.5.1'
-    implementation 'androidx.fragment:fragment-ktx:1.5.3'
+    implementation 'androidx.activity:activity-ktx:1.9.2'
```

Comment:
Is there an updated BOM we can use rather than specifying versions individually?

Reply:
As far as I know the BOM is only for Compose, I don't think it includes androidx activity

Reply:
Fair point - I thought we could grab a "Bill-of-Materials" for other, non-compose stuff so we could be confident it all works nicely together - but nope! Resolving.